test_conformance/api/test_kernels.cpp

```diff
@@ -704,3 +713,43 @@ REGISTER_TEST(negative_set_immutable_memory_to_writeable_kernel_arg)
 
     return TEST_PASS;
 }
+
+REGISTER_TEST(negative_invalid_arg_size)
+{
+    cl_int error = CL_SUCCESS;
```

Comment:
This test should only run when images are supported:

`PASSIVE_REQUIRE_IMAGE_SUPPORT(device)`

Reply:
Done